### DISCUSSION TITLE
List showing who you follow and who is following you

### DISCUSSION THREAD

**Discussion by Oxbowatos** [Traewelling/traewelling#2037] [Ideas / Feature Requests]
**Is your feature request related to a problem? Please describe.**
Idk if there is a list already but when yes I didn't found it.

**Describe the solution you'd like**
It would be nice to have a List which is easy accessible which shows which users I'm following and maybe an other list showing which users are following me, e. g. Twitter, Instagram or many other Socialmedia platforms.
For privacy it don't need to be public for other user to see who I'm following or who follows me.

**Comment by Oxbowatos** [comment]
Maybe also a list showing which user I blocked and muted

**Comment by vainamov** [comment]
Who follows you can be seen here: https://traewelling.de/settings/follower

**Comment by Oxbowatos** [comment]
Thanks @vainamov I didn't saw the option in the Settings maybe it's to Small.
I would prefer if its more prominent in the Page like a Bootstrap Button and that the Blocked users shouldn't be combined with the follower list and be its own list or rename the list.

**Comment by MrKrisKrisu** [comment]
Maybe we can add a "Followers" and "Following" link on the profile as Twitter does?
